Makefile

```diff
@@ -83,13 +84,14 @@ image-build: $(foreach i,$(IMAGE_TARGET_LIST),image/$(i)) ## Build all images.
 
 # Build an image.
 BUILD_IMAGE_REPO = quay.io/operator-framework
-image/%: BUILD_DIR = build/_image
-# Images run on the linux kernel, so binaries must always target linux.
-image/%: export GOOS = linux
-image/%: build/%
-	mkdir -p ./images/$*/bin && mv $(BUILD_DIR)/$* ./images/$*/bin
-	docker build -t $(BUILD_IMAGE_REPO)/$*:dev -f ./images/$*/Dockerfile ./images/$*
-	rm -rf $(BUILD_DIR)
+# When running in a terminal, this will be false. If true (ex. CI), print plain progress.
+ifneq ($(shell test -t 0; echo $$?),0)
+DOCKER_PROGRESS = --progress plain
+endif
+image/%: export DOCKER_CLI_EXPERIMENTAL = enabled
+image/%:
+	docker run --privileged --rm tonistiigi/binfmt:qemu-v5.0.1 --install all > /dev/null
```

Comment:
Do we need to do this setup, or will we only ever use this target to build images for the platform of the system its executed on? 

I ask because I think this make filesystem changes outside of the project directory, which would be good to avoid if possible.

Reply:
It's reasonable to assume that dev image builds will only be building for the host platform, so this can probably be removed.